### ISSUE TITLE
Allow creation dates with a length of 0

### ISSUE THREAD

**Issue by tagg7** [Radarr/Radarr#7926]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Some torrents contain a creation date key with a zero-length string as its value. Radarr will throw an exception if this is the case.

### Expected Behavior

The date should be ignored or set to epoch.

### Environment

```markdown
- OS: Linux
- Radarr: 4.3.2.6845
- Docker Install: Yes
- Using Reverse Proxy: 
- Browser: 
- Database:
```

### What branch are you running?

Master

### Trace Logs?

[v4.3.2.6836] MonoTorrent.TorrentException: Invalid torrent file specified
 ---> MonoTorrent.BEncoding.BEncodingException: Could not parse  into a number
 ---> System.FormatException: Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at MonoTorrent.Torrent.LoadInternal(BEncodedDictionary torrentInformation, InfoHash infoHash)
   --- End of inner exception stack trace ---
   at MonoTorrent.Torrent.LoadInternal(BEncodedDictionary torrentInformation, InfoHash infoHash)
   at MonoTorrent.Torrent.LoadCore(BEncodedDictionary torrentInformation, InfoHash infoHash)
   at MonoTorrent.Torrent.Load(Stream stream, String path)
   --- End of inner exception stack trace ---
   at MonoTorrent.Torrent.Load(Stream stream, String path)
   at MonoTorrent.Torrent.Load(Byte[] data)
   at NzbDrone.Core.MediaFiles.TorrentInfo.TorrentFileInfoReader.GetHashFromTorrentFile(Byte[] fileContents) in D:\a\1\s\src\NzbDrone.Core\MediaFiles\TorrentInfo\TorrentFileInfoReader.cs:line 14
   at NzbDrone.Core.Download.TorrentClientBase`1.DownloadFromWebUrl(RemoteMovie remoteMovie, String torrentUrl) in D:\a\1\s\src\NzbDrone.Core\Download\TorrentClientBase.cs:line 190
   at NzbDrone.Core.Download.TorrentClientBase`1.Download(RemoteMovie remoteMovie) in D:\a\1\s\src\NzbDrone.Core\Download\TorrentClientBase.cs:line 118
   at NzbDrone.Core.Download.DownloadService.DownloadReport(RemoteMovie remoteMovie) in D:\a\1\s\src\NzbDrone.Core\Download\DownloadService.cs:line 73
   at Radarr.Api.V3.Indexers.ReleaseController.DownloadRelease(ReleaseResource release) in D:\a\1\s\src\Radarr.Api.V3\Indexers\ReleaseController.cs:line 87
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Radarr.Http.Middleware.BufferingMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\BufferingMiddleware.cs:line 28
   at Radarr.Http.Middleware.IfModifiedMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\IfModifiedMiddleware.cs:line 41
   at Radarr.Http.Middleware.CacheHeaderMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\CacheHeaderMiddleware.cs:line 33
   at Radarr.Http.Middleware.UrlBaseMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\UrlBaseMiddleware.cs:line 27
   at Radarr.Http.Middleware.VersionMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\VersionMiddleware.cs:line 28
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)

[AB#4546](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/4546)

**Comment by Qstick** [comment]
Hi, this is already fixed in 4.2.4

**Comment by Qstick** [comment]
https://github.com/Radarr/Radarr/commit/e1f83c205d051b0ddd686a8da9a4075e7db14c11
